examples/small-study/src/background.js

```diff
@@ -98,6 +98,8 @@ class StudyLifeCycleHandler {
         // uninstall the addon?
         break;
     }
+    // actually remove the addon.
+    await browser.study.uninstall();
```

Comment:
any reason this isn't done in the `default` case above? That will run regardless of reason too right?